src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/Stage.component.js

```diff
@@ -14,13 +17,35 @@ const styles = {
 };
 
 
-export const StagePlain = ({ stage: { name, icon }, classes, className }: Props) => (
-    <div className={cx(classes.overview, className)}>
-        <StageOverview
-            title={name}
-            icon={icon}
-        />
-    </div>
-);
+export const StagePlain = (
+    { stage: { name, icon, stageForm },
+        events,
+        programStage,
+        classes,
+        className,
+    }: Props) => {
+    const [open, setOpenStatus] = useState(true);
+
+    return (<div className={cx(classes.overview, className)}>
+        <Widget
+            header={<StageOverview
+                title={name}
+                icon={icon}
+                description={stageForm.description ?? ''}
+                events={events}
+            />}
+            onOpen={useCallback(() => setOpenStatus(true), [setOpenStatus])}
+            onClose={useCallback(() => setOpenStatus(false), [setOpenStatus])}
+            open={open}
+        >
+            {events.length > 0 && <StageDetail
```

Comment:
Hey, if there are no events the widget will be empty?

Reply:
hey @simonadomnisoru , yes based on the [mockup](https://www.sketch.com/s/95c3d586-218c-4685-968b-a388270da949/a/DvrbKa/play) the widget is empty